kas/include/mender-base.yml

```diff
@@ -20,7 +20,7 @@ repos:
 
   meta-mender:
     url: https://github.com/mendersoftware/meta-mender.git
-    commit: 4302cb71a7f8a0ffb09f47e41f4ce1df3e812419
+    commit: 21b8bd76e92391665ad2cf52fa1def65ba97c7a2
```

Comment:
needs bump to e8b6b3e554392f6d0af3eb18667d93e658205250 so your PR on persistent machine IDs is already merged